pkg/image/api/v1/register.go

```diff
@@ -27,6 +27,11 @@ func init() {
 		&ImageStreamImage{},
 		&ImageStreamImport{},
 	)
+	api.Scheme.AddDefaultingFuncs(
```

Comment:
Add later in another PR if needed? It's ok to leave it if you want, but it might confuse people as to why it's here and empty.